src/lib.rs

```diff
@@ -420,10 +416,42 @@ macro_rules! derive_alias {(
                 )}
                 #[allow(unused_imports)]
                 pub(in crate) use [< $MacroName __derive_macro >] as $MacroName;
-            )*
-        }
-    )}
-)}
+            }
+        )}
+    };
+    (pub #[derive($MacroName:ident !)] = #[derive($($derives:tt)*)];) => {
```

Comment:
Let's add support for docstrings as well, then 🙂 

```suggestion
    (
        $( #[doc $($doc:tt)*] )*
        pub #[derive($MacroName:ident !)] = #[derive($($derives:tt)*)];
    ) => (
```